apps/progs.pl

```diff
@@ -95,15 +95,20 @@
         pkcs12   => "des",
     );
     my %cmd_deprecated = (
-        rsa      => [ "3_0", "pkey",      "rsa" ],
-        genrsa   => [ "3_0", "genpkey",   "rsa" ],
-        rsautl   => [ "3_0", "pkeyutl",   "rsa" ],
-        dhparam  => [ "3_0", "pkeyparam", "dh" ],
-        dsaparam => [ "3_0", "pkeyparam", "dsa" ],
-        dsa      => [ "3_0", "pkey",      "dsa" ],
-        gendsa   => [ "3_0", "genpkey",   "dsa" ],
-        ec       => [ "3_0", "pkey",      "ec" ],
-        ecparam  => [ "3_0", "pkeyparam", "ec" ],
+# The format of this table is:
+#   [0] = 0/1, 1 means deprecated and gone, 0 is deprecated but still present
```

Comment:
This makes zero sense.  Deprecated stuff will be removed (at least publicly) some time in the future.  Adding another middle ground is just confusing.

Reply:
This is so that the commands get a "this command is deprecated" message printed when run.  The leading 0/1 will go away in all likelihood (once they are all updated).